# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1338494697 14400
#      Thu May 31 16:04:57 2012 -0400
# Branch stable
# Node ID b9bbd4c569baffdbfefe6ffe7ac44108dbd16f62
# Parent  551566201318bf615b27c60ccf9368f4844008bd
Added tag release-3-6-2 for changeset 551566201318

diff --git a/.hgtags b/.hgtags
--- a/.hgtags
+++ b/.hgtags
@@ -57,8 +57,9 @@ 3781981be535e80d44c85373b8fdaa60ca5cd097
 ff5588774680d4f54567311fc109c8e351950f1c ss-3-5-91
 a737b3fb9c4d89d3694da6b4e623aeee64b212e1 ss-3-5-92
 72aebe6196414e38ef802469ff6e238f914f04c9 rc-3-6-0-0
 64d9f33313cc8c691974bcd123357e24bccbabdc rc-3-6-0-1
 704f7895eef03008dd79848eb9da4bfb40787d73 release-3.6.0
 95c43fc8dbe1a07a46fefb3372df5b2309d874fd rc-3-6-1-0
 f947d2922febf12dcd1fb6e21b356756ecb54e55 rc-3-6-2-0
 4460c4fb20e6a5d3b1972fa737d4e00eb921545a rc-3-6-2-2
+551566201318bf615b27c60ccf9368f4844008bd release-3-6-2
